### DISCUSSION TITLE
Gempy 3 in google colab or jupyter notebook?

### DISCUSSION THREAD

**Discussion by lucianomarquetto** [gempy-project/gempy#923] [Q&A] [answered]
Hi Gempy Community! I am curious about this: is Gempy runnable in google colab or still better to use jupyter notebook/lab? I have been using the old (2.3) version in jupyter notebook, but now want to learn version 3 and wanted to check if there is an updated response comparing to https://github.com/orgs/gempy-project/discussions/714 (from 2022). 

Really excited to try version 3, congratulations to all involved!

Cheers,

Luciano.

**Comment by javoha** [comment]
Hi,
the short answer is: Yes - gempy v3 runs on google colab. There are some issues with the pyvista 3D visualization but at least static plots should work.
Cheers,
Jan

**Comment by lucianomarquetto** [comment]
Hi Jan,

Thanks a lot for your response! I will keep working on jupyter lab, but will give colab a try for static plots too.

Cheers!

Luciano

**Comment by flohorovicic** [comment, marked as answer]
HI @lucianomarquetto ,

I set up an example model in Google Colab which should still work: 

[Google Colab GemPy](https://colab.research.google.com/drive/1JhXMJrGrJeXJK2MOB-btrEb_eyA8hk3o?usp=sharing)

Hope it works - and if you create one, feel free to share with us as well (if you can :-) ).

Kind regards, Florian

**Reply by lucianomarquetto** [reply]
Hi @flohorovicic,

Thanks a lot for the example, I will try it and if I have any issue will come back here!

Cheers,

Luciano.
